### PR TITLE
Add command for tilesets activity

### PR DESCRIPTION
**Pull request by branyip**
## Details

- Added a command to list activity data for the requesting user's tilesets
- Some misc updates to make Travis happy

To-do:
- [ ] `git tag -a v1.9.0 -m 'v1.9.0' && git push --tags`
- [ ] Confirm release at https://pypi.org/project/mapbox-tilesets/#history

## Tests
_Installed locally_

### Basic command success
image
image

### Unauthorized failures
image
image

### Invalid argument value failures
image
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,3 +403,19 @@ tilesets tilejson <tileset_id>
 Flags:
 
 * `--secure`: By default, resource URLs in the retrieved TileJSON (such as in the "tiles" array) will use the HTTP scheme. Include this query parameter in your request to receive HTTPS resource URLs instead.
+
+### list-activity
+
+List tileset activity for an account. Returns a pagination key `next` if there are more results than the return limit that can be passed into another command as the `start` argument.
```
Comment (jprittie) [inline README.md:409]: Hi there - we know what activity means, but could we clarify it for users? i.e. total requests for a tileset over the past 30 days.

File: mapbox_tilesets/scripts/cli.py [hunk 2]
```diff
@@ -799,3 +800,78 @@ def estimate_area(features, precision, no_validation=False, force_1cm=False):
...
+def list_activity(
+    username,
+    sortby=None,
+    orderby=None,
+    limit=None,
+    start=None,
+    token=None,
+    indent=None,
+):
+    """List tileset activity for an account. Response is an ordered array of aggregate resource data.
```
Comment (jprittie) [inline mapbox_tilesets/scripts/cli.py:845]: I'm not sure "an ordered array of aggregate resource data" will make sense to users. Could we say what it's ordered by, and then just turn "aggregate resource data" into plainer language? i.e. sum of requests over the past 30 days.

**Review by jprittie — COMMENTED** [review]
I know it's not in scope for this PR, but since we're here, could we print-user friendly messages instead of tracebacks when a request is unauthorized?

**Comment by branyip** [comment]
@jprittie Good call to clarify the documentation, I've made some updates. As discussed earlier we'll leave the stack traces for a separate ticket since it seems to be the existing pattern for tilesets-cli

**Review by jprittie — APPROVED** [review]

**Pull request merged**